### PR TITLE
feat: Initial payload for EtherFi Ethereum activation

### PR DESCRIPTION
**Pull request by kartojal**
AIP proposal to activate Aave v3 EtherFi Ethereum instance following [[ARFC] Deploy an Etherfi/Stablecoin Aave v3 Instance](https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440)

### CODE REVIEW

File: src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+title: "EtherFi Ethereum Activation"
+author: "Catapulta"
+discussions: "https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x4acd11c6100a6b85a553e21359f3720fa5cd4783a76c77857436ace134f88c05"
+---
+
+## Simple Summary
+
+This AIP activates the EtherFi and Stables Aave V3 instance in Ethereum network, following [ARFQ request](https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440).
```
Comment (eboadom) [inline src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md:10]:
```suggestion
This AIP activates an Aave v3 EtherFi instance in the Ethereum network, following [ARFQ request](https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440), focused on weETH holders borrowing stablecoins.
```

File: src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
+---
+title: "EtherFi Ethereum Activation"
```
Comment (eboadom) [inline src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md:2]:
```suggestion
title: "Aave v3 EtherFi Ethereum Activation"
```

File: src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  - Listing of weETH, USDC, PYUSD, and FRAX tokens.
+- Post execution:
+  - Add PROTOCOL_GUARDIAN as POOL_ADMIN role
+  - Add CAPS_PLUS_RISK_STEWARD as RISK_ADMIN role
+  - Seed amounts
+  - Pay Catapulta service fee
+
+The table below illustrates the configured risk parameters for **weETH**
+
+| Parameter                 |                                      Value |
```
Comment (brotherlymite) [inline src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md:40]: @kartojal i would add all the assets in a single table as its a bit easier to read

Comment (kartojal) [reply src/20240902_AaveV3EthereumEtherFi_EtherFiEthereumActivation/EtherFiEthereumActivation.md]: done!

**Review by eboadom — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**